### PR TITLE
Feature/platfowner/global path

### PR DESCRIPTION
**Pull request by platfowner**
This change was for supporting global paths in transactions.
Although it's not including all the features needed for the purpose, let me try to merge it asap as it includes refactoring parts that may affect other incoming CLs for sharding.

Summary:
- Add TokenProperties, AccountProperties
- Let DB has sharding path
- Apply some renaming

#60

### CODE REVIEW

File: constants.js [hunk 1]
```diff
@@ -268,9 +292,11 @@ function getGenesisConfig(filename, additionalEnv) {
 function getGenesisSharding() {
   const config = getGenesisConfig('genesis_sharding.json');
   if (config[ShardingProperties.SHARDING_PROTOCOL] === ShardingProtocols.POA) {
-    ChainUtil.setJsObject(config, [ShardingProperties.SHARD_OWNER], GenesisAccounts.owner.address);
-    ChainUtil.setJsObject(
-        config, [ShardingProperties.SHARD_REPORTER], GenesisAccounts.others[0].address);
+    const ownerAddress = ChainUtil.getJsObject(
+          GenesisAccounts, [AccountProperties.OWNER, AccountProperties.ADDRESS]);
+    const reporterAddress = GenesisAccounts[AccountProperties.OTHERS][0][AccountProperties.ADD];
```
Comment (liayoo) [inline constants.js:297]: ADD => ADDRESS ?

Comment (platfowner) [reply constants.js]: Fixed. Thanks!

**Review by liayoo — APPROVED** [review]
LGTM!

**Review by minsulee2 — APPROVED** [review]
All checked! LGTM

**Comment by platfowner** [comment]
Thanks for the quick review!
Now merging...

**Pull request merged**